Nasal/engine.nas

```diff
@@ -329,6 +350,8 @@ controls.adjMixture = func {
     setprop("/controls/engines/current-engine/mixture", new_value);
 };
 
+# backwards compatibility only - the controls.throttleAxis should not be overridden like this. The joystick binding Throttle (all) has 
```

Comment:
The `master` branch is expected to be used with fgdata git and will be copied to fgdata when is going to be released 2018.2, so I guess the backwards compatibility stuff can be removed.

Reply:
This is runtime backwards compatibility for existing Input/joysticks/xml files that haven't been rebound to use the new methods. Once you change a binding with the joystick dialog it will of course fix that entry; but otherwise old or manually editted joystick XML files will not work.

Reply:
Could you add a comment that it's needed for old joystick XML files?